src/main/java/carracing/model/CarRacing.java

```diff
@@ -1,40 +1,32 @@
-package carracing;
+package carracing.model;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class CarRacing {
     private static final int MAX_POSITION_INITIAL_VALUE = 0;
-    private static final int RACING_COUNT_FIRST_INDEX = 0;
     private List<Car> cars;
+    private List<String> winners;
 
     public CarRacing(List<Car> cars) {
         this.cars = cars;
     }
```

Comment:
이 부분은 아래와 같이 변경해볼 수 있을 것 같아요.

```suggestion
    public CarRacing(List<Car> cars) {
        this.cars = new ArrayList<>(cars);
    }
```
외부에서 `CarRacing`에 입력해준 `cars`에 접근할 경우 발생할 수 있는 상황에 대해 고민해보면 좋을 것 같네요! :)